cloud_governance/tag_cluster/remove_cluster_tags.py

```diff
@@ -120,19 +134,35 @@ def get_cluster(self, clusters: list):
                                 break
         return [cluster_dict, cluster_tags]
 
-    def cluster_instance(self):
+    def cluster_update_tags(self, cluster: list, queue):
```

Comment:
queue type ?

Reply:
It is a Multithread Queue. So I didn't know the exact type